Source/WebKit/Platform/ios/PlaybackSessionInterfaceLMK.mm

```diff
@@ -196,6 +202,16 @@ - (void)linearMediaPlayer:(WKSLinearMediaPlayer *)player setVideoReceiverEndpoin
         model->setVideoReceiverEndpoint(videoReceiverEndpoint);
 }
 
+- (void)_exitFullscreen
+{
+    auto model = _model.get();
+    if (!model)
+        return;
+
+    model->toggleFullscreen();
```

Comment:
And it seem equally odd to call `toggleFullscreen` from `_exitFullscreen`

Reply:
The reasoning is similar here; since we only call `-_exitFullscreen` when we are already in fullscreen mode, toggling is expected to exit fullscreen. I'll make this more explicit by introducing an `exitFullscreen()` and calling that instead.

Reply:
Actually I added a `FIXME` for now. Hopefully that's sufficient for now (albeit not ideal) given the check added above.